IDENTITY_USE_CASES.md

```diff
@@ -0,0 +1,98 @@
+# Identity Use Cases in OpenSearch
+- [Identity Use Cases in OpenSearch](#identity-use-cases-in-opensearch)
+  - [Non-use compatibility](#non-use-compatibility)
+    - [Scenario 1:](#scenario-1)
+  - [Identity features enabled](#identity-features-enabled)
+    - [Scenario 2:](#scenario-2)
+    - [Scenario 3:](#scenario-3)
+  - [Using Admin account](#using-admin-account)
+    - [Scenario 4:](#scenario-4)
+    - [Scenario 5:](#scenario-5)
+  - [Password changes](#password-changes)
+    - [Scenario 6:](#scenario-6)
+    - [Scenario 7:](#scenario-7)
+    - [Scenario 8:](#scenario-8)
+  - [Authenticate as new user](#authenticate-as-new-user)
+    - [Scenario 9:](#scenario-9)
+    - [Scenario 10:](#scenario-10)
+    - [Scenario 11:](#scenario-11)
+  - [Multi-node Cluster](#multi-node-cluster)
+    - [Scenario XXX:](#scenario-xxx)
+
+## Non-use compatibility
+
+    Given: Start an OpenSearch cluster with Identity experiment enabled and not configured
+
+### Scenario 1:
+
+The cluster runs with the Security plugin. No Security plugin scenarios are impacted.
+
+*Technical*: Identity uses the NoopAuthenticationManager
+
+## Identity features enabled
+
+    Given: Start an OpenSearch cluster with Identity experiment enabled, internal identity is enabled in the configuration
+
+### Scenario 2:
+
+By reading the opensearch.log / console output a random admin password is shown.
+
+### Scenario 3:
+
+All REST API activity returns 403 without passing authentication information in the request
+
+## Using Admin account
+
+    Given: The admin password is known
+
+### Scenario 4:
+
+Using http basic auth it is possible to make any REST API request, no errors related to authentication are seen.
+
+### Scenario 5:
+
+Admin makes a request to `PUT /identity/user/admin/password` with a json body `{ password: "value" }`, it succeeds
+
+*Technical*: Need user actions
+
+## Password changes
+
+    Given: The Admin password has been updated
+
+### Scenario 6:
+
+The admin's old password does not work on any REST API requests
```

Comment:
I think that this is supposed to read as a result of the password change. Is this formatting of the different results separated into scenarios common? I would tend to think of scenarios as being sort of separate from one another but this is basically a result of scenario 5.

Reply:
Maybe this is overly verbose, but we should check that password updates are immediately reflected.  If this is a format for which we are defining 'acceptance criteria' I think this might be meaningful, but happy to accept updates where we trim down the scenarios.